packages/lesswrong/components/ultraFeed/UltraFeedSettings.tsx

```diff
@@ -381,33 +464,63 @@ const UltraFeedSettings = ({
     
     setZodErrors(null);
     
-    updateSettings(result.data);
+    updateSettings(result.data as Partial<UltraFeedSettingsType>);
```

Comment:
This cast should be unnecessary given that it's the direct output of zod parsing